tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -75,8 +75,8 @@ Status ModularFileSystem::NewWritableFile(
 }
 
 Status ModularFileSystem::NewAppendableFile(
-    const std::string& fname,
-    std::unique_ptr<WritableFile>* result /*, TransactionToken* token */) {
+    const std::string& fname, std::unique_ptr<WritableFile>* result,
+    TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `result`.